app/models/exercise_assignment.rb

```diff
@@ -2,13 +2,14 @@
 #
 # Table name: exercise_assignments
 #
-#  id           :bigint           not null, primary key
+#  id           :integer          not null, primary key
 #  completed_at :datetime
-#  data         :jsonb
+#  data         :json
+#  due_by       :datetime
```

Comment:
Hm, this seems significant. Is this added for the front-end portion of the test?

Reply:
I switched us from pg -> sqlite to make setup easier. Should apply across the board.

I don't think we're doing anything in the pairing interview where pg is valuable or sqlite will get in the way, but happy to discuss if that's wrong.

Reply:
Oh, you meant `due_by`. We _might_ use it if time permits, but it's not mission-critical. I might leave it for now, but we can remove it if we don't end up using it.